docs/guide/usage/autocompletion.md

```diff
@@ -0,0 +1,25 @@
+name: autocompletion
+category: usage
+tags: guide
+index: 8
+title: WebdriverIO - Autocompletion
+---
+
+Autocompletion
+=====================
+
+If you have been writing program code for a while, you probably like autocompletion.
+Autocomplete is available out of the box in many code editors.
+However if autocompletion is required for some packages that are not installed in the usual locations, these too could be added via some configuration changes.
+
+There'a two ways to add support for our framework:
+
+1. Specify the location where commands are placed: `<your_package>/node_modules/webdriverio/lib/commands`.
```

Comment:
Could you be more specific here? I believe autocompletion does not work out of the box with webdriverio. I thought this would be a good chance to add documentation around how to setup autocompletion for different ids. Since you are using WebStorm could you give a more detailed description how to set that up for the WebStorm IDE. Thing about someone has a blank WebStorm version installed on his computer and wants to have WebdriverIO autocompletion.